scripts/concordanceCheck_array-array.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -eu
+
+concordanceDir="/groups/umcg-atd/tmp06/concordance/array-array/"
+
+#
+##
+### Needs an index folder with a file to compare with
+### Data to compare with should be in ${concordanceDir}
+### Creates subfolders automatically
+### data will be bgzipped/tabix'ed
+### only first 100K lines will be selected for comparison
+### output will be in results ${concordanceDir}/results/
+##
+#
+
+echo "creating workfolders : results,tmp,samplesheets,jobs and original in ${concordanceDir}"
+mkdir -p "${concordanceDir}/"{results,tmp,samplesheets,jobs,original}
+
+module load "HTSlib/1.3.2-foss-2015b"
```

Comment:
As this script is for debugging potential sample swaps manually, maybe it is better not to used hard-coded versions and do report on which version was used for the comparison.
```suggestion
module load HTSlib
module list
```